serverless.env.yaml

```diff
@@ -0,0 +1,10 @@
+vars: null
```

Comment:
Oh yes, that's an old file (see: https://serverless.com/framework/docs/providers/aws/guide/variables#migrating-serverlessenvyml).

This could be replaced with another `.yml` file which is referenced or removed altogether (if not needed).